services/nillion-interactor/programs/muscle-perform.py

```diff
@@ -0,0 +1,24 @@
+from nada_dsl import *
+
+def nada_main():
+
+    party1 = Party(name="Party1")
+
+
+    snp = SecretInteger(Input(name="snp", party=party1))
+    genotype = SecretInteger(Input(name="genotype", party=party1))
+
+    target_snp = Integer(6)
+    target_genotype = Integer(12)
```

Comment:
You shouldn't copy the logic exactly from `thrombosis.py`. Here we are testing for a different condition, i.e. muscle performance which looks at the SNP `rs1815739` and the genotype `TT` to identify endurance athletes.

Reply:
Changed thrombosis mapping to 12th SNP and 7th genotype ie `rs6025` and `GG`.
for muscle performance changed mapping to 11th SNP and 9th genotype ie `rs1815739` and `TT`.